scripts/uberenv/devtools.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Am I reading this right that uberenv now directly supports devtools?

Reply:
Uberenv supports it based on giving it another uberenv project file.  This PR ties the devtools together by pointing the serac package at the previously built devtools in the packages.yaml file.  It also allows you to build the whole TPL+devtools in one fell swoop, though I dont recommend it.